categories.py

```diff
@@ -52,6 +52,7 @@
   "clacaputo":        ["reconstruction"],
   "kskovpen":         ["pdmv"],
   "sunilUIET":        ["pdmv"],
+  "miquork":          ["pdmv"],
```

Comment:
@perrotta , `miquork` is already defined at https://github.com/cms-sw/cms-bot/blob/bc8a9cd1465a2d7128a18a09edc7066ac3de9ddd/categories.py#L121 , please remove it from here and add `"pdmv"` at https://github.com/cms-sw/cms-bot/blob/bc8a9cd1465a2d7128a18a09edc7066ac3de9ddd/categories.py#L121

Reply:
thank you, I did not notice!